builtin/app/ruby/data/common/dev/Vagrantfile.tpl

```diff
@@ -134,6 +146,20 @@ oe gem install bundler --no-document
 ol "Bundling gem dependencies..."
 oe bundle
 
+{% if app_type == "rails" %}
+  ol "Detected Rails application"
+
+  if has_gem pg; then
```

Comment:
One way this falls short: If the `pg` gem is in the production group, but we use Sqlite for local development. Not sure if this is worth worrying about, or what the solution would be.